### PR TITLE
Bind target script names

### LINKED ISSUE #410: in house test infrastructure allowed confusing divergence between script and make target names

### PR DESCRIPTION
**Pull request by zancas**
@pjbreaux
#### What issues does this address?
Fixes #410

### CODE REVIEW

File: systest/Makefile [hunk 1]
```diff
@@ -175,27 +175,28 @@ tempest_12.1.1_undercloud:
...
+	$(MAKE) -C . run_tests
 
 # Once we have the variables setup we can run the tempest tests on our env
-tempest_tests:
-	$(MAKE) -C . tempest_tests-setup
-	$(MAKE) -C . tempest_tests-config
-	$(MAKE) -C . tempest_tests-run
-	$(MAKE) -C . tempest_tests-cleanup
+run_tests:
+	$(MAKE) -C . tlc_session_setup
```
Comment (pjbreaux) [inline systest/Makefile]: Why did you break away from the '-' delimiter?

File: systest/Makefile [hunk 2]
```diff
@@ -175,27 +175,28 @@ tempest_12.1.1_undercloud:
...
-tempest_tests-run:
+neutron_lbaas_run:
 	@echo "running the tempest test cases..."
-	cd scripts && ./tempest_tests.sh || $(MAKE) -C ../ tempest_tests-cleanup
+	cd scripts && ./$@.sh || $(MAKE) -C ../ tlc_session_cleanup
 
-tempest_tests-cleanup:
+tlc_session_cleanup:
 	@echo "cleaning up down the TLC session..."
 	-cd scripts && ./tlc_session_cleanup.sh
```
Comment (pjbreaux) [inline systest/Makefile]: Automatic variable should be applicable here as well.

Comment (zancas) [inline systest/Makefile]: Good point!

File: systest/Makefile (hunk 1 above)
Comment (zancas) [inline systest/Makefile]: Good question: To make filenames not have '-' in them.

**Pull request merged**